com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/HDRaytracingLightCluster.cs

```diff
@@ -293,14 +293,16 @@ void BuildGPULightVolumes(HDRayTracingLights rayTracingLights)
                     // Reserve space in the cookie atlas
                     m_RenderPipeline.ReserveCookieAtlasTexture(currentLight, light, currentLight.type);
 
-               
+                    // Compute the camera relative position
+                    Vector3 lightPositionRWS = currentLight.gameObject.transform.position - hdCamera.camera.transform.position;
```

Comment:
there should be a if here like iin lightloop.cs, we do the relative only if camera relative is enabled, so the shader  code don't need to have lot of ifdef with GetAbsolutePosition